app/states/texas/texas_downloader.py

```
@@ -7,90 +9,271 @@
-
-
-        return self
-
-    def read(self) -> RecordGen:
-        _reader = FileReader()
-        self.data = _reader.read_folder(self.folder, file_counts=self.config.FILE_COUNTS)
-        return iter(self.data)
+        if _safe_to_delete:
+            _task1 = progress.add_task("T1", "Removing existing files", "In Progress")
+            ic()
+            list(map(lambda x: x.unlink(), _existing_files))
```

Ruff (0.14.7)

Unnecessary `map()` usage (rewrite using a list comprehension)

Replace `map()` with a list comprehension

(C417)